docs/AboutUs.md

```diff
@@ -1,52 +1,33 @@
 # About Us
 
-We are a team based in the [School of Computing, National University of Singapore](http://www.comp.nus.edu.sg).
+We are a team in the [CS 2103 Class](http://www.comp.nus.edu.sg/~cs2103/AY1617S1/).
 
 ## Project Team
 
-#### [Damith C. Rajapakse](http://www.comp.nus.edu.sg/~damithch) <br>
-<img src="images/DamithRajapakse.jpg" width="150"><br>
-**Role**: Project Advisor
+#### [Nishant Budhdev](https://github.com/nishantbudhdev) <br>
+<img src="images/Nishant Budhdev.jpg" width="150"><br>
+**Role**: Project Mentor
 
 -----
 
-#### [Joshua Lee](http://github.com/lejolly)
-<img src="images/JoshuaLee.jpg" width="150"><br>
-Role: Developer <br>  
-Responsibilities: UI
+#### [Murray Chen](https://github.com/mchen14) <br>
+Role: Team Lead, Developer <br>
+Responsibilities: Code Quality
 
 -----
 
-#### [Leow Yijin](http://github.com/yijinl) 
-<img src="images/LeowYijin.jpg" width="150"><br>
+#### [Zihua Huang](https://github.com/JamesHuangUC)
 Role: Developer <br>  
-Responsibilities: Data
+Responsibilities: Integration, Git Expert, Scheduling and Tracking
 
 -----
 
-#### [Martin Choo](http://github.com/m133225)
-<img src="images/MartinChoo.jpg" width="150"><br>
+#### [Yaquan Wang](https://github.com/A0116137M) 
 Role: Developer <br>  
-Responsibilities: Dev Ops
-
------
-
-#### [Thien Nguyen](https://github.com/ndt93)
- Role: Developer <br>  
- Responsibilities: Threading
- 
- -----
-
-#### [You Liang](http://github.com/yl-coder) 
-<img src="images/YouLiang.jpg" width="150"><br>
- Role: Developer <br>  
- Responsibilities: UI
+Responsibilities: Documentation, Testing, Deliverables and Deadlines
```

Comment:
You can also mention the packages each person is in-charge of in this file. If not, I hope you have already divided it and have an understanding of the same among yourself.